packages/vuetify/src/util/rebuildFunctionalSlots.ts

```diff
@@ -1,10 +1,12 @@
+import { CreateElement, VNodeChildren } from "vue";
+
 /**
  *
  * @param {object} slots
  * @param {function} h
  * @returns {array}
  */
-export default function rebuildFunctionalSlots (slots, h) {
+export default function rebuildFunctionalSlots (slots: Record<string, VNodeChildren>, h: CreateElement) {
```

Comment:
in vue.d.ts $slots is defined as `readonly $slots: { [key: string]: VNode[] | undefined };`, so maybe it should be the same here